### DISCUSSION TITLE
cant open game

### DISCUSSION THREAD

**Discussion by Imejeli** [skyline-emu/skyline#1698] [Q&A] [answered]
cant open game Pokemon arceus on poco F3..hm,please advice.

**Comment by mateoalfaro** [comment]
Incompatible title and please don't spam the same issue

**Comment by Dekuzearou** [comment]
That game doesn't work please just wait

**Comment by Crytonics** [comment, marked as answer]
Game doesn't work yet.
You can follow updates here https://github.com/skyline-emu/skyline-games-list/issues/247
